ee/codegen/src/__test__/nodes/__snapshots__/search-node.test.ts.snap

```diff
@@ -37,6 +37,14 @@ class SearchNodeDisplay(BaseSearchNodeDisplay[SearchNode]):
     label = "Search Node"
     node_id = UUID("search")
     target_handle_id = UUID("370d712d-3369-424e-bcf7-f4da1aef3928")
+    metadata_filter_input_id_by_operand_id = {
+        UUID("a6322ca2-8b65-4d26-b3a1-f926dcada0fa"): UUID(
+            "c539a2e2-0873-43b0-ae21-81790bb1c4cb"
+        ),
+        UUID("c539a2e2-0873-43b0-ae21-81790bb1c4cb"): UUID(
+            "c539a2e2-0873-43b0-ae21-81790bb1c4cb"
+        ),
+    }
```

Comment:
We were hitting a failure case previously that caused us to not codegen these ids when we should have been